examples/solid/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@example/solidstart",
+  "scripts": {
+    "dev": "solid-start dev --port 3000",
```

Comment:
Can we change the port? If we don't, `pnpm dev` will be broken in the repo by default